jcl/src/jdk.management/share/classes/openj9/lang/management/OpenJ9DiagnosticsMXBean.java

```diff
@@ -68,10 +68,20 @@
 	 */
 	public void resetDumpOptions() throws ConfigurationUnavailableException;
 
+	/**
+	 * Returns the current dump configuration as an array of Strings.
+	 * The syntax of the option Strings is the same as the -Xdump command-line option,
+	 * with the initial -Xdump: omitted. See the -Xdump option section on dump agents in
+	 * the documentation for the OpenJ9 JVM.
+	 *
+	 * @throws SecurityException if there is a security manager and it doesn't allow the checks required to read the dump settings
+	 */
+	public String[] queryDumpOptions() throws InvalidOptionException, ConfigurationUnavailableException;
```

Comment:
The possibility of `InvalidOptionException` should be documented.

Reply:
I forgot to remove these.